scripts/make-docs.js

```diff
@@ -0,0 +1,38 @@
+var execp = require('./execp');
+var fs = require('fs');
+
+module.exports = function () {
+  var script = "docgen/target/docgen.js";
+  var api = "derivable.api.edn";
+  var tsOut = "dist/derivable.d.ts";
+  var htmlOut = "index.html";
+
+  if (!fs.existsSync("docs")) {
+    fs.mkdirSync("docs");
+  }
+
+  var rootQ = null;
+  if (fs.existsSync(script)) {
+    console.log("docgen exists, skipping boot");
+    rootQ = {then: function (cb) { return cb(); }};
+  } else {
+    console.log("doicgen doesn't exist, compiling it...");
+    rootQ = execp("boot cljs", "./docgen");
```

Comment:
this seems to require the cljs "boot" tool. Is it possible to avoid having to have the global "boot" installed?